packages/headless/src/controllers/insight/facets/range-facet/date-facet/headless-insight-date-filter.test.ts

```diff
@@ -0,0 +1,99 @@
+import {buildMockDateFacetRequest} from '../../../../../test/mock-date-facet-request';
```

Comment:
These tests were copied from `headless-date-filter.test.ts` keeping only tests related to dispatched actions.